setup.py

```diff
@@ -107,16 +142,17 @@
       maintainer=__maintainer__,
       maintainer_email=__email__,
       url='http://www.biom-format.org',
-      test_suite='nose.collector',
       packages=find_packages(),
+      tests_require=['pytest', 'pytest-cov'],
```

Comment:
can you add the missing test's dependencies in this list i.e. nose and flake8?

Reply:
Done